services/jupyterhub/jupyterhub-k8s.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM jupyterhub/k8s-hub:v0.6
+FROM rrrrrok/k8s-hub:8aab19b
```

Comment:
Switch back to official one?
Or still culling issues?

Reply:
good point, the PR has been merged to master